packages/twitch/src/API/Helix/HypeTrain/HypeTrainEvent.ts

```diff
@@ -0,0 +1,84 @@
+import { Enumerable } from '@d-fischer/shared-utils';
+import { ApiClient } from '../../../ApiClient';
+import { HypeTrainContributor } from '../HypeTrain/HypeTrainContributor';
+
+/** @private */
+export interface HypeTrainEventData {
+	eventId: string;
+	broadcasterId: string;
+	level: string;
+	startedAt: string;
+	expiresAt: string;
+	total: string;
+	lastContribution: HypeTrainContributor;
```

Comment:
Did you mean `HypeTrainContributorData`? You're also gonna need to properly convert the data to class instances in the getters.